### DISCUSSION TITLE
Implementing avamb workflow

### DISCUSSION THREAD

**Discussion by johnne** [metagenome-atlas/atlas#636] [Ideas]
Hi,
I like the addition of vamb as a binner. Have you seen the [Avamb Snakemake workflow](https://github.com/RasmussenLab/vamb/tree/master/workflow_avamb) by the vamb developers? I wonder if it's something you plan on implementing in atlas?

Thanks for a nice tool!

**Comment by SilasK** [comment]
What exactly is the difference between vamb and avamb?

**Comment by johnne** [comment]
Main difference seems to be the type of "encoder" used in the neural network. I admit the underlying method is too technical for me to grasp fully, but while the original vamb uses a "variational autoencoder", the new binner (called AAMB) uses an "adversarial autoencoder" (see the biorxiv paper [here](https://www.biorxiv.org/content/10.1101/2023.02.27.527078v1.full)) which the authors claim perform even better than vamb. The Avamb workflow combines both tools in order to generate better bins compared to running each tool separately.

**Reply by SilasK** [reply]
I looked at it and the workflow seems not so straight forward to copy. Have you tried it?
